scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -199,6 +199,22 @@ static struct node_ctx *lookup_node_ctx(u32 node)
 	return nodec;
 }
 
+struct {
+	__uint(type, BPF_MAP_TYPE_ARRAY);
+	__type(key, int);
+	__type(value, struct dsq_ctx);
+	__uint(max_entries, LO_FALLBACK_DSQ);
+	__uint(map_flags, 0);
+} dsq_data SEC(".maps");
+
+static struct dsq_ctx *lookup_dsq_ctx(u64 dsq_idx)
+{
+	struct dsq_ctx *dsqc;
+
+	dsqc = bpf_map_lookup_elem(&dsq_data, &dsq_idx);
```

Comment:
Does it want to trigger error when lookup fails? Otherwise, it can just be `return bpf_map_lookup_elem(..)`

Reply:
Yeah, probably should error out.